terra-jupyter-base/jupyter_notebook_config.py

```diff
@@ -0,0 +1,51 @@
+# adapted from https://github.com/jupyter/docker-stacks/blob/master/base-notebook/jupyter_notebook_config.py
```

Comment:
This file is currently in Leo, see comment: https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/ClusterHelper.scala#L485-L488

I'll also make the Leo version consistent with this one, as I had to change a few things.

Reply:
It doesn't seem like there's much risk if they diverge, but it may be worth putting a comment at the top mentioning an identical file lives in another repo